package.json

```diff
@@ -19,13 +19,22 @@
     "build:cli": "yarn workspace @web3api/cli build",
     "build:client-js": "yarn workspace @web3api/client-js build",
     "build:wasm-ts": "yarn workspace @web3api/wasm-ts build",
+    "lint:code": "lerna run lint:code",
+    "lint:code:ci": "lerna run --parallel lint:code",
     "test": "yarn test:wasm-ts && yarn test:client-js",
     "test:wasm-ts": "yarn workspace @web3api/wasm-ts test",
     "test:client-js": "yarn workspace @web3api/client-js test",
     "test:ci": "yarn test:wasm-ts && yarn test:ci:client-js",
     "test:ci:client-js": "yarn workspace @web3api/client-js test:ci"
   },
   "devDependencies": {
-    "lerna": "3.22.1"
+    "@typescript-eslint/eslint-plugin": "^4.9.0",
```

Comment:
Can we use fixed versions wherever possible (no `^`)? I've been bit a number of times by minor version number changes introducing breaking changes :P

Reply:
Yes, of course!